src/zzz_od/gui/view/hollow_zero/withered_domain_setting_interface.py

```
@@ -162,11 +162,15 @@ def on_interface_shown(self) -> None:
 
     def _update_run_record_display(self) -> None:
         if self.run_record.period_reward_complete:
-            content = '已完成刷取周期性奖励 如错误可重置'
+            content = gt('已完成刷取周期性奖励 如错误可重置')
```

Ruff (0.16.1)

[error] Undefined name `gt`

(F821)